components/admin_console/team_channel_settings/team/list/team_list.jsx

```diff
@@ -96,6 +101,35 @@ export default class TeamList extends React.PureComponent {
         this.loadPage(0, term);
     };
 
+    onFilter = (filterOptions) => {
+        const filters = {};
+        const {group_constrained, allow_open_invite, invite_only} = filterOptions.management.values; // eslint-disable-line camelcase, @typescript-eslint/camelcase
+        const allowOpenInvite = allow_open_invite.value; // eslint-disable-line camelcase, @typescript-eslint/camelcase
+        const inviteOnly = invite_only.value; // eslint-disable-line camelcase, @typescript-eslint/camelcase
+        const groupConstrained = group_constrained.value; // eslint-disable-line camelcase, @typescript-eslint/camelcase
+        const filtersList = [allowOpenInvite, inviteOnly, groupConstrained];
```

Comment:
You can also do...
```js
/*eslint-disable camelcase, @typescript-eslint/camelcase */
const {group_constrained, allow_open_invite, invite_only} = filterOptions.management.values;
const allowOpenInvite = allow_open_invite.value;
const inviteOnly = invite_only.value;
const groupConstrained = group_constrained.value;
/*eslint-disable camelcase, @typescript-eslint/camelcase */
```
...to disable the rule for everything between the comments.